src/Symfony/Component/Lock/Tests/Store/SemaphoreStoreTest.php

```diff
@@ -50,13 +50,22 @@ public function testResourceRemoval()
 
     private function getOpenedSemaphores()
     {
+        if ('Darwin' === PHP_OS) {
+            $lines = explode(PHP_EOL, trim(`ipcs -s`));
```

Comment:
Note: I'm not sure using `LC_ALL=C` as in mentioned PR would change anything on OS X.

Reply:
Changing the locale doesn't change the `ipcs`output, fine as is to me